src/screens/Apps/Escrow/RpcHandler.ts

```
@@ -0,0 +1,183 @@
+}
+type Result = { tag: 'success'; result: OutboundMessage } | { tag: 'failure'; error: Error }
+export default function makeMessageHandler(props: Props) {
+  return async function messageHandler(message: InboundMessage): Promise<Result> {
+    switch (message.kind) {
+      case 'ARKADE_KEEP_ALIVE':
+        return { tag: 'success', result: { kind: 'ARKADE_KEEP_ALIVE', timestamp: Date.now() } }
+      case 'ARKADE_RPC_REQUEST': {
+        const { id, method } = message
+        switch (method) {
+          case 'get-x-public-key':
+            const xOnlyPublicKey = await props.getXOnlyPublicKey().then((k) => (k ? bytesToHex(k) : null))
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)